.gitignore

```diff
@@ -9,3 +9,4 @@
 !/etc/build/.gitkeep
 
 /tests/Application/yarn.lock
+/.idea
```

Comment:
Please, remove it from the .gitignore. Configure a global .gitignore file instead. Read more here: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c